mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -247,9 +246,22 @@ def benchmarkMIOpen(commandLine, outputFile, xdlops):
         nanoSeconds = 0
     output(config.generateCSVContent(nanoSeconds), outputFile)
 
+def geo_mean(data):
```

Comment:
Why geometric mean? I feel like arithmetic mean makes better sense because each profiling is a independent events.

Reply:
I think geometric mean can better represent the overall performance, because it reduces the impact of the outliers. I can also add arithmetic mean if we think it helps.

Reply:
Sounds good. I think for the time being just geo mean should serve us. We can decide whether we'd need more later.